test/config/config.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     port: env.SEQ_MSSQL_PORT || env.SEQ_PORT || 1433,
     dialectOptions: {
       options: {
+        encrypt: false,
```

Comment:
Why? Is this leftover from #12300?

Reply:
This is for local testing, So MSSQL docker test can work locally without installing certs